web/src/base_page_params.ts

```diff
@@ -47,6 +47,7 @@ const home_params_schema = default_params_schema
         narrow_stream: z.optional(z.string()),
         needs_tutorial: z.boolean(),
         promote_sponsoring_zulip: z.boolean(),
+        realm_rendered_description: z.optional(z.string()),
```

Comment:
We should add a comment above this line noting that this parameter is only sent for spectators, because it isn't displayed for logged-in users.

Reply:
Done